meta-dts-distro/recipes-kernel/kernel/linux-yocto/efivars.cfg

```diff
@@ -1 +1 @@
-CONFIG_EFI_VARS=y
```

Comment:
resolves Dasharo/dts-scripts#29
This change is also requiere if this is sufficient